include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Sorry, looks like I missed one of your earlier messages.

> I don't know about cmake, but... when you install the apt-get package or the homebrew package, at the moment the config.h that is generated by autoconf is NOT installed yet. This has been something to-do as it would require renaming the config.h to something else (as the filename config.h is a bit too easy to lead to mistakes) and then prefix the configure results and install it.

If so, that seems like a packaging error.  The config.h file is just as much a part of the required headers for the devel package as any of the others, since the user won't be running configure again.

It doesn't need renaming as it is only included by a file in the same directory so cannot possibly conflict with anything else in CppUTest itself, and shouldn't conflict with anything else that is built with CppUTest in the include path since they should have similar local conventions (at least on anything POSIX-like).

It shouldn't be included in source packages, of course -- for those the user is expected to run `configure` themselves (though not `autoreconf`, which is why `configure` is included even though it's a generated file).